README.md

```diff
@@ -1,74 +1,270 @@
+
+Note that you only need to specify a host if you are on a dedicated Kinvey instance. Otherwise just press Enter to continue.
+
+When prompted for `Profile`, enter a name for your new working profile that Kinvey CLI will create for you. Kinvey CLI will use this profile automatically for future executions as long as it is the only profile on the system. You can create new profiles and select an active account if you need to.
+
 ```
-kinvey config -e kinveyAccount@kinvey.com -p yourKinveyPassword
+$ kinvey init
+? E-mail john.doe@kinvey.com
+? Password ***********
+? Host https://manage.kinvey.com/
+? Profile dev
 ```
 
-#### Environment variables
-* `KINVEY_CLI_USER` - e-mail address of your Kinvey account
-* `KINVEY_CLI_PASSWORD` - password of your Kinvey account
+You can run `kinvey init` from any directory as it always writes you new profile in your home directory.
+
+Next, you need to configure Kinvey CLI to connect to a Flex Service that you've already created using the Kinvey Console.
+
+For the following commands, you need to switch to the Node.js project directory that you will be deploying as a Flex Service as the configuration they create and read is project-specific.
 
-Example:
 ```
-export KINVEY_CLI_USER=kinveyAccount@example.com && export KINVEY_CLI_PASSWORD=yourKinveyPassword && kinvey config
+cd <node.js project dir>
+kinvey flex init
 ```
 
-### Proxy Settings
+Through a series of prompts, this command will ask you for a domain in which to operate (app or organization) and a Flex Service to deploy to.
+
+Finally, you are ready to deploy you node.js project as a Flex Service.
+
+    kinvey flex deploy
+
+**Note**: Kinvey CLI sends binary data (content type "multipart/form-data") during the deploy process. The deploy job will fail if traffic of this type is blocked within your network.
+
+## Managing Profiles
+
+If you still haven't run `kinvey init` or if you want to create a second, third, and so on profile, you can do so by invoking `kinvey profile create <name>`. You can choose between providing the credentials at the command line or as preset [environment variables](#environment-variables).
 
-The Kinvey CLI supports the universal ENV variables `HTTPS_PROXY` and `https_proxy` for routing commands through a proxy server. For example:
+`kinvey profile create dev --email john.doe@kinvey.com --password john'sPassword --host kvy-us2`
 
+You can create multiple profiles and specify which one to use at the command line.
+
+`kinvey flex init --profile dev`
+
+If you don't want to specify a profile every time, you can set one as active and it will be used for future executions:
+
+`kinvey profile use dev`
+
+
+**Note**: If you have a single profile, you can skip setting it as active or providing it as command line option. It will be used if no other credentials are provided.
```

Comment:
"active or providing it as command line option" - maybe as **a** command line

Reply:
👍